lib/dispatcher.rb

```diff
@@ -10,11 +10,18 @@ def email_observers(cart)
     end
   end
 
+  def email_already_approved_with_changes(cart)
+    cart.approved_approvals.each do |approval|
+      CommunicartMailer.cart_notification_email(approval.user_email_address, approval, false).deliver
+    end
+  end
+
   def email_sent_confirmation(cart)
     CommunicartMailer.proposal_created_confirmation(cart).deliver
   end
 
   def deliver_new_cart_emails(cart)
```

Comment:
It's currently happening in the linear/parallel dispatchers. https://www.pivotaltracker.com/story/show/90215472 is an effort to make this more consistent